xamarin/Xamarin.Forms
# [Android/iOS] Fix inconsistent behavior when setting StrokeDashArray

### Description of Change ###

Fix inconsistent behavior when setting **StrokeDashArray**.

### Issues Resolved ### 

- fixes #12203 

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS
- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

#### Before
Crash using a null value in Android.

#### After
<img width="486" alt="Captura de pantalla 2020-09-25 a las 11 44 50" src="https://user-images.githubusercontent.com/6755973/94255304-518a1500-ff28-11ea-82c5-615ea4f14d8f.png">

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 11203. Verify all the Ellipses render in the same way on Android, iOS, etc.

### PR Checklist ###
